tests/hbacsvcgroup/test_hbacsvcgroup_client_context.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Test hbacsvcgroup
+  hosts: ipaclients
+  become: true
+  gather_facts: false
+
+  tasks:
+  - name: Include FreeIPA facts.
+    include_tasks: ../env_freeipa_facts.yml
+
+  # Test will only be executed if host is not a server.
+  - name: Execute with server context in the client.
+    ipahbacsvcgroup:
+      ipaadmin_password: SomeADMINpassword
+      ipaapi_context: server
+      name: ThisShouldNotBeAdded
```

Comment:
Side note: There are `name: ThisShouldNotWork`, `name: ThisShouldNotBeAdded` and `name: THIS_SHOULD_FAIL`. Are there different templates for this file?

Reply:
Fixed.